src/main/java/reiff/projectile/Projectile.java

```diff
@@ -39,5 +45,9 @@ public double getInterceptX() {
         double timeToGround = (2 * velocity * Math.sin(radians)) / 9.8;
         return Math.cos(radians) * velocity * timeToGround;
     }
+
+    public double getTotalTime() {
+        return (2 * velocity * Math.sin(radians)) / 9.8;
```

Comment:
getApexTime() * 2

Reply:
What is this referring to?

Reply:
Take a look at the code for this method and the code for getApexTime